src/Symfony/Component/DomCrawler/Tests/Html5ParserCrawlerTest.php

```diff
@@ -46,6 +46,20 @@ public function testHtml5ParserWithInvalidHeadedContent(string $content)
         self::assertEmpty($crawler->filterXPath('//h1')->text(), '->addHtmlContent failed as expected');
     }
 
+    public function testHtml5ParserNotSameAsNativeParserForSpecificHtml()
+    {
+        // Html who create a bug specific to the DOM extension (see https://github.com/symfony/symfony/issues/28596)
```

Comment:
⚠️ A test that relies on the presence of a bug to pass is extremely brittle. If the purpose of the test is to ensure the parser is different then that is what the assertion should be checking, even if you have to use reflection to achieve it.